compiler/util/llvmDebug.cpp

```diff
@@ -124,14 +125,14 @@ LLVM_DITYPE debug_data::construct_type(Type *type)
   const char* name = type->symbol->name;
   ModuleSymbol* defModule = type->symbol->getModule();
   const char* defFile = type->symbol->fname();
-  if (strstr(defFile, "/modules/")!=NULL || strcmp(defFile, "<internal>")==0) {
+  /*if (strstr(defFile, "/modules/")!=NULL || strcmp(defFile, "<internal>")==0) {
 #if HAVE_LLVM_VER >= 37
     return NULL;
 #else
     return llvm::DIType();
 #endif
 
-  }
+  }*/
```

Comment:
Ah, you are right, I recall that it was me that suggested those strcmps in order to make llvm 3.7 work in the last PR. But unfortunately, this condition will disable the appropriate debugging generation of variables as well. So it has to be removed for llvm 3.3. But llvm 3.7 isn't working well for this change when -g is added in compilation.